plugin/storage/cassandra/schema/docker.sh

```diff
@@ -1,33 +0,0 @@
-#!/bin/bash
-#
-# This script is used in the Docker image jaegertracing/jaeger-cassandra-schema
-# that allows installing Jaeger keyspace and schema without installing cqlsh.
```

Comment:
this comment is critical. I would prefer not to delete this file, and I don't think it's necessary to do given my objections above about combining C server with C-schema image. The latter is a public API of Jaeger release.

Reply:
Got it, will fix as stated above.